packages/oscal-react-library/src/stories/OSCALProperties.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { StoryFn } from "@storybook/react";
+import { Property } from "@easydynamics/oscal-types";
+import { OSCALPropertiesDialog, OSCALPropertiesDialogProps } from "../components/OSCALProperties";
+
+const exampleProperties: Property[] = [
+  {
+    name: "Example Property 1",
+    value: "Example Value",
+    ns: "http://easydynamics.com/ns/oscal/example",
+  },
+  {
+    name: "Example Property 2",
+    value: "Example Value",
+  },
+  {
+    name: "Example Property 3",
+    value: "Example Value",
+  },
+];
```

Comment:
For our storybook content, would we rather include our data within where we're defining the story or keep all data to a seperate file?

Reply:
i like this better

Reply:
I think for something so small and specific to the `Properties` storybook page, it is ok to put it in the file itself.